core/src/navigation/ContextSwitcher.html

```diff
@@ -0,0 +1,184 @@
+      </button>
+    </div>
+    <div class="fd-popover__body" aria-hidden="{dropDownStatesNegated.CONTEXT_SWITCHER_POPOVER || true}" id="CONTEXT_SWITCHER_POPOVER">
+      <nav class="fd-menu">
+        {#if actions && actions.length}
+        <ul class="fd-menu__list fd-menu__list--top">
+          {#each actions as node}
+          {#if node.position === 'top' || !['top', 'bottom'].includes(node.position)}
+          <li><a class="fd-menu__item" on:click="onActionClick(node)">{node.label}</a></li>
+          {/if}
+          {/each}
+        </ul>
+        {/if}
+
+        <ul class="fd-menu__list" id="context_menu_middle">
+        {#if options && options.length === 0}
+          <li><div class="fd-spinner"><div></div></div></li>
+        {/if}
+        {#if options && options.length}
+          {#each options as node}
+          <li><a class="fd-menu__item" on:click="goTo(node.path)">
+          {#if node.label}{node.label}{/if}
+          {#if !node.label}
+            {#await getFallbackNodeName(config.fallbackLabelResolver, node.id)}
+            {:then label}
+              {label}
+            {/await}
+          {/if}
+          </a></li>
+          {/each}
+        {/if}
+        </ul>
+        {#if actions && actions.length}
+        <ul class="fd-menu__list fd-menu__list--bottom">
+          {#each actions as node}
+          {#if node.position === 'bottom'}
+          <li><a class="fd-menu__item" on:click="onActionClick(node)">{node.label}</a></li>
+          {/if}
+          {/each}
+        </ul>
+        {/if}
+      </nav>
+    </div>
+  </div>
+</div>
+{/if}
+
+<script type="text/javascript">
+  import { ContextSwitcherHelpers } from './services/context-switcher.js';
+  import { LuigiConfig } from '../services/config.js'
+  import * as Helpers from '../utilities/helpers.js';
+  import * as Routing from '../services/routing.js';
+
+  export default {
+    async oncreate() {
+      const contextSwitcherConfig = LuigiConfig.getConfigValue('navigation.contextSwitcher');
+      this.set({
+        contextSwitcherEnabled: !!contextSwitcherConfig,
+        config: contextSwitcherConfig
+      });
+
+      if (contextSwitcherConfig) {
+        const actions = await LuigiConfig.getConfigValueAsync('navigation.contextSwitcher.actions');
+        const currentPath = Routing.getCurrentPath();
+
+        this.set({
+          actions,
+          fallbackLabelResolver: contextSwitcherConfig.fallbackLabelResolver
+        });
+
+        // options are loaded lazy by default
+        if (ContextSwitcherHelpers.isContextSwitcherDetailsView(currentPath, contextSwitcherConfig.parentNodePath)
+            || contextSwitcherConfig.lazyloadOptions === false) {
+          await this.fetchOptions();
+          await this.setSelectedContext(currentPath);
+        }
+      }
+
+      Routing.addRouteChangeListener(() => this.setSelectedContext(Routing.getCurrentPath()));
+    },
+    data() {
+      return {
+        dropDownStates: {},
```

Comment:
Context switcher dropdown and logout dropdown don't work together very well. 
* open context switcher dropdown
* close context switcher dropdown
* open logout dropdown -> context switcher dropdown opened as well

Reply:
Fixed in https://github.com/kyma-project/luigi/pull/190/commits/6509c2ed1e67daf3a437e95c6fdfb8fda5683b9b

Reply:
Clearly a bug of _blur_ toggle, which we wanted to simplify.